src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -360,6 +363,15 @@ private IQueryable<Instance> QueryBuilderForEnded(IQueryable<Instance> queryBuil
             return queryBuilder.Where(i => i.Process.Ended == dateValue);
         }
 
+        private IQueryable<Instance> QueryBuilderExcludeConfirmedBy(IQueryable<Instance> queryBuilder, string queryValue)
+        {
+            return queryBuilder.Where(i =>
+                i.CompleteConfirmations == null ||
```

Comment:
Cosmos isnt really a fan of asserting null. Ref the bug we had processComplete=false.
I don't think you'll get any matches, so maybe we could just stick to the second clause?

Reply:
Removing it does doesn't cause any errors or changes the result in my local testing with CosmosDB emulator. So I'll do just that. Doing the reverse caused the query to exclude instances I would have expected to see. In other words, the null check is never true.